src/index.ts

```diff
@@ -27,7 +27,7 @@ async function run() {
     arch = 'aarch64'
   }
 
-  const url = `https://releases.captain.build/${version}/${os}/${arch}/captain${extension}`
+  const url = `https://github.com/rwx-research/captain/releases/download/${version}/captain-${os}-${arch}${extension}`
```

Comment:
What if we left releases.captain.build but updated the cloudfront function to redirect to the github URL? Then we could still collect metrics on downloads?

Reply:
That's a good idea 👍